src/main/java/net/javadiscord/javabot2/config/guild/ModerationConfig.java

```diff
@@ -16,6 +16,31 @@
 	 */
 	private long staffRoleId;
 
+	/**
+	 * The amount of seconds to be looked into the past to determine if a user is spamming.
+	 */
+	private int pastMessageCountBeforeDurationInSeconds;
+
+	/**
+	 * The amount of messages to be sent within {@link #pastMessageCountBeforeDurationInSeconds}.
+	 */
+	private int messageSpamAmount;
```

Comment:
This description doesn't really make sense. Is the bot sending messages, or users? What happens when this amount is reached? Also preferably spam-related variables should start with `spam` so that it's easier to identify them when looking through all moderation settings.